docs/tutorials/create_architecture.md

```diff
@@ -1,221 +1,225 @@
-# 評価準備
+# Preparation for measuring latency of callback-chains
 
-CARET は、評価する際にアーキテクチャファイルの作成が必要です。
+CARET is capable of measuring latency of not only callback functions and topic communications, but also node chains.
+Specifying a callback function as target is very simple because name is corresponding to the callback function. Topic communication is specified by a topic name, a source node, and a destination node. However, specifying a node chain is more difficult than them. A node chain is defined as a combination of callback functions and topic communications, and often called as "path". Though node chain is regarded as DAG (Direct Acyclic Graph), a node chain may have loops and forks. Loops and forks make a node chain make complicated. Additionally, a large system tends to have many candidates of node chains because it is constructed of many callback functions which are communicated via many topics.
 
-トレース結果からアーキテクチャファイルの雛形を作成できますが、一部手作業での修正が必要になります。  
-ここでは、アーキテクチャファイルの作成方法とその修正方法を説明します。
+## Overview of an architecture file
 
-## アーキテクチャファイルとは
+To deal with such difficulty of specifying node chains, CARET serves configuration for users to choose target node chains. The configuration is defined in a yaml file, which is called "**architecture file**". An architecture file describes a target application's structure. CARET can create an architecture file for minimum description automatically with using measured data, but it does not have target node chains. You will add description for target node chains to a generated architecture file.
 
-アーキテクチャファイルとは、実際に実行されたノードの情報など、測定毎に不変な静的な情報のみを含んだ yaml 形式の設定ファイルです。  
-コールバックの実行時間などの動的な情報は含みません。  
-※ トピック名やタイマーの周期など、実行時のパラメータも一部含みます。
+The rest of this section will explain more details of an architecture file.
 
-アーキテクチャファイルで記述した静的な情報と、  
-コールバックの実行時間などの測定結果と合わせることで、レイテンシの算出が行えるようになります。
+## What to describe in an architecture file
+
+An architecture file is a yaml-based file which has description of an application measured by CARET. In particular, an architecture file includes information representing the application structure, such as executors, nodes, callback groups, callbacks, topics, timers and so on. It can be used for another trial measurement because such information shall be same unless the application is changed.
 
 ---
 
-アーキテクチャファイルの主な役割は以下の通りです。
+User have to add the following description to architecture file.
+
+1. Target node chains and its structure
+2. Intra-communication of topic in a single node to calculate latency of the node
```

Comment:
1. static information (e.g. executors, nodes, callback)
2. latency definition

Reply:
Here, I'd like to mention what users have to add in an architecture file. Static information and latency definition are main portion of the architecture file, but they are not added by users.

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/d63db973ff51f0af897bef167dd4fe3ccab4c7c9